WordPress/Classes/Services/KeyValueStorage.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+protocol KeyValueStorage {
+    func valueForKey(key: String) -> AnyObject?
+    func setValue(value: AnyObject, forKey key: String)
```

Comment:
Looking at `NSUserDefaults.setObject(_:forKey:)` documentation:

> The value parameter can be only property list objects: NSData, NSString, NSNumber, NSDate, NSArray, or NSDictionary. For NSArray and NSDictionary objects, their contents must be property list objects. See What is a Property List? in Property List Programming Guide.

Considering the expected usage, I don't think it's worth trying to serialize other types, but the protocol documentation should list the supported types.